plugins/inputs/dpdk/sample.conf

```diff
@@ -14,20 +14,37 @@
   # socket_access_timeout = "200ms"
 
   ## Enables telemetry data collection for selected device types.
-  ## Adding "ethdev" enables collection of telemetry from DPDK NICs
-  ## (stats, xstats, link_status).
-  ## Adding "rawdev" enables collection of telemetry from DPDK Raw Devices
-  ## (xstats).
+  ## Adding "ethdev" enables collection of telemetry from DPDK NICs (stats, xstats, link_status, info).
+  ## Adding "rawdev" enables collection of telemetry from DPDK Raw Devices (xstats).
   # device_types = ["ethdev"]
 
   ## List of custom, application-specific telemetry commands to query
   ## The list of available commands depend on the application deployed.
   ## Applications can register their own commands via telemetry library API
-  ## http://doc.dpdk.org/guides/prog_guide/telemetry_lib.html#registering-commands
+  ## https://doc.dpdk.org/guides/prog_guide/telemetry_lib.html#registering-commands
   ## For L3 Forwarding with Power Management Sample Application this could be:
   ##   additional_commands = ["/l3fwd-power/stats"]
   # additional_commands = []
 
+  ## List of plugin options.
+  ## Supported options:
+  ##  - "dpdk_in_memory" option enables reading for multiple sockets when a dpdk application is running with --in-memory option.
+  ##    When option is enabled plugin will try to find additional socket paths related to provided socket_path.
+  ##    Details: https://doc.dpdk.org/guides/howto/telemetry.html#connecting-to-different-dpdk-processes
+  # plugin_options = ["dpdk_in_memory"]
+
+  ## Specifies plugin behavior regarding unreachable socket (which might not have been initialized yet).
+  ## Available choices:
+  ##   - error: Telegraf will return an error during the startup and gather phases if socket is unreachable
+  ##   - ignore: Telegraf will ignore error regarding unreachable socket on both startup and gather
+  # unreachable_socket_behavior = "error"
```

Comment:
Are those backward compatible?

Reply:
Yes, if plugin_options will be empty

Reply:
What about the socket behaviour? What did telegraf do before?

Reply:
unreachable_socket_behavior = "error" - represents the same behavior as prev. versions (the plugin will fail if there is no socket)

Reply:
Okay, then it is all fine by me.